### PR TITLE
LPS-126301

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case is already covered by ExportImport#ExportImportWebcontentWithRepeatField

**Comment by alexcandidos** [comment]
Hi @Tim-Cao , thanks for the review! Discussing with @emanuelfisilva , we'll keep them as is in the Data Engine scope.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase [hunk 1]
```diff
@@ -53,12 +53,56 @@ definition {
...
+			fieldName = "Text"
+		);
+
+		DataEngine.addFieldNested(
+			fieldFieldLabel = "Numeric",
+			fieldName = "Numeric",
+			targetFieldLabel = "Text"
+		);
+
+        DataEngine.addFieldNested(
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase:86]: This step is adding a nested fields group

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3629
View total diff https://github.com/liferay-echo/liferay-portal/pull/3629/commits/20b9fe66cc7011741ba04868db13fb2f56173ee3

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**